.github/workflows/issue_type_predicter.yml

```diff
@@ -89,7 +89,7 @@ jobs:
       - name: Run Issues Type Predicter
         if: env.analysis_response == 'Valid'
         run: |
-          echo "is_wish_list="$(dotnet run -p ./IssuesTypePredicter/IssuesTypePredicterML.ConsoleApp/IssuesTypePredicterML.ConsoleApp.csproj -v q "${{ env.issue_json_string }}")"" >> $GITHUB_ENV
+          echo "is_wish_list="$(dotnet run --project ./IssuesTypePredicter/IssuesTypePredicterML.ConsoleApp/IssuesTypePredicterML.ConsoleApp.csproj -v q "${{ env.issue_json_string }}")"" >> $GITHUB_ENV
```

Comment:
@avidit What the reason for this change?

Reply:
We are getting this warning message `Warning NETSDK1174: The abbreviation of -p for --project is deprecated. Please use --project.` during the run
e.g. https://github.com/DynamoDS/Dynamo/actions/runs/9991315977/job/27613858963#step:11:8

More on the warning message https://learn.microsoft.com/en-us/dotnet/core/tools/sdk-errors/netsdk1174